bin/helm-bump-edge

```diff
@@ -1,34 +1,52 @@
 #!/usr/bin/env bash
 
-# This script bumps the patch version of all charts whose versions end in -edge.
-# The BUMP_CRDS and BUMP_CNI variables can be set to also bump those versions.
+# This script bumps$the patch version of all charts
 
 set -euo pipefail
 shopt -s globstar
 
 bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
 
-# Iterate through charts/*/Chart.yaml and bump the patch version of the
-# `version` field. E.g. 1.18.2-edge becomes 1.18.3-edge.
+# shellcheck source=_tag.sh
+. "$bindir"/_tag.sh
+tag=$(named_tag)
+
+edge_tag_regex="edge-([0-9][0-9])\.([0-9]|[0-9][0-9])\.([0-9]+)"
+
+# Get the current edge version.
+url=https://run.linkerd.io/install-edge
```

Comment:
Given that the edge- tag already specifies the edge version, do we need to recompute the new version independently? How do we ensure that the computed value and the actual tag don't differ?

Reply:
It's not necessary to recompute the version but this avoids making mistakes. A few lines below it's ensured the tag and the computed version match.